Gems/ROS2/gem.json

```
@@ -13,9 +17,20 @@
     "user_tags": [
         "ROS2"
     ],
+    "compatible_engines": [
+        "o3de-sdk==1.2.0",
+        "o3de>=2.0.0"
```

It looks like somehow the 23.05.0 o3de SDK engine name was not `o3de-sdk` in the installers, so I recommend lowering the `o3de` engine version requirement to 1.2.0 - I would leave the `o3de-sdk==1.2.0` in there in case we release a point fix that corrects this.

```suggestion
        "o3de>=1.2.0"
```